crates/bevy_a11y/src/lib.rs

```diff
@@ -7,10 +7,10 @@
 
 //! Accessibility for Bevy
 
-use std::sync::{
-    atomic::{AtomicBool, Ordering},
-    Arc,
-};
+extern crate alloc;
+
+use alloc::sync::Arc;
+use core::sync::atomic::{AtomicBool, Ordering};
```

Comment:
In cases where the `alloc` crate is required, an `extern crate alloc;` must be placed in the `lib.rs`/`main.rs` file at the crate root. It may be possible to remove these if we instead created a `bevy_alloc` crate, and depend on it for the `alloc` types. This may be desirable anyway, as it would be a more appropriate home for things like `HashMap` and `EntityHashSet`.

Reply:
I'd rather us just follow convention instead of establishing our own patterns. This seems like a rather non-intrusive requirement.

Reply:
I agree, and there's definitely a benefit to keeping it explicit and simple, especially considering this feature is aimed at the more low-level side of Rust programming.